lib/Dialect/EmitC/Conversion/TosaToEmitC.cpp

```diff
@@ -72,6 +72,55 @@ class CallOpConversion : public OpConversionPattern<SrcOp> {
   bool explicitOperandTypes;
 };
 
+class Conv2DOpConversion : public OpConversionPattern<tosa::Conv2DOp> {
+
+public:
+  Conv2DOpConversion(MLIRContext *ctx) : OpConversionPattern(ctx) {}
+
+private:
+  LogicalResult
+  matchAndRewrite(tosa::Conv2DOp convOp, ArrayRef<Value> operands,
+                  ConversionPatternRewriter &rewriter) const override {
+    // fail if quantization is requested
```

Comment:
`// fails`

Reply:
I chose the imperative because I think it fits better here. It is supposed to fail! You have to read it with a drill sergeant voice in your mind ;)

Reply:
Sir yes sir! Ok fine by me